dashboard/test/ui/runner.rb

```diff
@@ -321,7 +321,9 @@ def output_synopsis(output_text)
 
     message += "<br/><i>rerun: ./runner.rb -c #{browser_name} -f #{feature} --html</i>"
     HipChat.log message, color: 'red'
-    HipChat.developers short_message, color: 'red' if CDO.hip_chat_logging
+    if CDO.hip_chat_logging && !(['adhoc', 'production'].include? Rails.env)
```

Comment:
Note: The hip_chat_logging check in hip_chat.rb does not disable posting messages to Slack:

https://github.com/code-dot-org/code-dot-org/blob/staging/lib/cdo/hip_chat.rb#L40

If we're okay with that, I'll make this change.